schemas/x509.yml

```diff
@@ -0,0 +1,154 @@
+      type: keyword
+      description: Common name (CN) of issuing certificate authority.
+      example: DigiCert SHA2 High Assurance Server CA
+
+    - name: issuer.organizational_unit
+      level: extended
+      type: keyword
+      description: List of organizational units (OU) of issuing certificate authority.
+      example: www.digicert.com
+
+    - name: issuer.organization
+      level: extended
+      type: keyword
+      description: Organizational units (O) of issuing certificate authority.
+      example: DigiCert Inc
+
+    - name: issuer.locality
+      level: extended
+      type: keyword
+      description: Locality name (L)
+      example: Mountain View
+
+    - name: issuer.state_or_province
+      level: extended
+      type: keyword
+      description: State or province name (ST, S, or P)
+      example: California
+
+    - name: issuer.country
+      level: extended
+      type: keyword
+      description: Country (C) code
+      example: US
+
+    - name: signature_algorithm
+      level: extended
+      type: keyword
+      description: Identifier for certificate signature algorithm. Recommend using names found in Go Lang Crypto library (https://golang.org/src/crypto/x509/x509.go).
+      example: SHA256-RSA
+
+    - name: valid_date
+      level: extended
+      type: date
+      description: Time at which the certificate is first considered valid.
+      example: 2019-08-16T01:40:25Z
+
+    - name: expire_date
+      level: extended
+      type: date
+      description: Time at which the certificate is no longer considered valid.
+      example: 2020-07-16T03:15:39Z
+
+    - name: subject.distinguished_name
+      level: extended
+      type: keyword
+      description: Distinguished name (DN) of the certificate subject entity.
+      example: C=US, ST=California, L=San Francisco, O=Fastly, Inc., CN=r2.shared.global.fastly.net
+
+    - name: subject.common_name
+      level: extended
+      type: keyword
+      description: Common name (CN) of subject.
+      example: r2.shared.global.fastly.net
+
+    - name: subject.organizational_unit
+      level: extended
+      type: keyword
+      description: List of organizational units (OU) of subject.
+
+    - name: subject.organization
+      level: extended
+      type: keyword
+      description: Organizational units (O) of subject.
+      example: Fastly, Inc.
+
+    - name: subject.locality
+      level: extended
+      type: keyword
+      description: Locality name (L)
+      example: San Francisco
+
+    - name: subject.state_or_province
+      level: extended
+      type: keyword
+      description: State or province name (ST, S, or P)
+      example: California
+
+    - name: subject.country
+      level: extended
+      type: keyword
+      description: Country (C) code
+      example: US
+
+    - name: public_key_algorithm
+      level: extended
+      type: keyword
+      description: Algorithm used to generate the public key.
+      example: RSA, ECDSA
```

Comment:
Is this an array, or just two different examples?

If it's meant as two different examples, please only pick one. The docs will make it look like the example value is the combination of these two, which may lead to confusion.

Reply:
yes, sorry. I'll pick one